source/usage-examples/find.txt

```diff
@@ -0,0 +1,70 @@
+.. _kotlin-usage-find:
+
+=======================
+Find Multiple Documents
+=======================
+
+.. default-domain:: mongodb
```

Comment:
This setting doesn't do anything anymore, so we can remove from all files (not in scope but just noting)